terraform/gcp/modules/sigstore/sigstore.tf

```diff
@@ -184,6 +182,9 @@ module "fulcio" {
   // Certificate authority
   ca_pool_name = var.ca_pool_name
 
+  // KMS
+  fulcio_keyring_name = "fulcio-keyring"
+  fulcio_key_name     = "fulcio-key"
```

Comment:
Is this the section on declaring variable values? If so, can this be `fulcio-intermediate-key`?

Reply:
sure!